transport/stream.go

```diff
@@ -20,14 +20,13 @@ import (
 	"net"
 )
 
-// StreamConn is a net.Conn that allows for closing only the reader or writer end of
-// it, supporting half-open state.
+// StreamConn is a [net.Conn] that allows for closing only the reader or writer end of it, supporting half-open state.
```

Comment:
[StreamConn]

Reply:
Type names should not be wrapped in `[]` in the documentation that describes the type itself.